ansible/roles/tilemaker/templates/bicycle/style.json

```diff
@@ -2348,6 +2348,24 @@
         "line-width": {"base": 1.3, "stops": [[15, 0.5], [20, 6]]}
       }
     },
+    {
+      "id": "poi-bike-track",
+      "type": "line",
+      "source": "openmaptiles",
+      "source-layer": "transportation",
+      "minzoom": 15,
+      "filter": [
+        "all",
+        ["==", "class", "track"],
+        ["any", ["in", "tracktype", "grade3", "grade2", "grade1"], ["==", "mtb:scale", 0]]
```

Comment:
Can you make this a bit more abstract? Please extract a subtype so that you can write a simpler selector here like `subtype=bicycle`?